### PR TITLE
CEM-347 horizontal scroll list

### PR DESCRIPTION
**Pull request by indancisive**
HorizontalScrollList3

@ralph-dev

### CODE REVIEW

File: src/Containers/HorizontalScrollList.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+export interface ScrollListDivProps {
+    menuWidth?: number;
+}
+
+export interface ListItemProps {
+    label: string;
+    hoveredStyle: Function;
+    selectedStyle: Function;
```
Comment (indancisive) [inline src/Containers/HorizontalScrollList.tsx:24]: Unsure if styling should be passed as a function that returns a string, as a standalone string, or as some other data type. Looked into React.CSSProperties but was unable to figure anything out

**Comment by indancisive** [comment]
@ralph-dev @fpunny
HorizontalScrollList4
Please take a look at the two style props. Right now they are functions that return strings. Unsure if they should be just strings or something else entirely for better maintainability (possibly React.CSSProperties ?)

File: src/Containers/HorizontalScrollList.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                <DropDownDiv menuWidth={menuWidth}>
+                    <Select
+                        placeholder={menuName}
+                        onChange={({ target }: { target: any }) => {
+                            setSelectedLabel(target.value);
+                        }}
+                        value={displaySelected ? selectedLabel : menuName}
+                    >
+                        {labelArray.map(label => (
+                            <option value={label}>{label}</option>
```
Comment (ralph-dev) [inline src/Containers/HorizontalScrollList.tsx:51]: Where is the key prop?

File: src/Containers/HorizontalScrollList.tsx [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                </DropDownDiv>
+            )}
+            <HorizontalList>
+                {labelArray.map((label, index) => (
+                    <HorizontalListItem
+                        key={label + index}
+                        onClick={() => setSelectedLabel(label)}
+                        label={label}
+                        hoveredStyle={
+                            hoveredStyle ? hoveredStyle : defaultHoveredStyle
```
Comment (ralph-dev) [inline src/Containers/HorizontalScrollList.tsx:63]: this is a CSS thing not a JS thing, put an isHovering Prop and Style it in the CSS styles component.

Comment (ralph-dev) [reply src/Containers/HorizontalScrollList.tsx:63]: Or just use isHover from CSS?

**Comment by ralph-dev** [comment]
Approve, resolve conflicts first

**Pull request merged**